src/constants/DefaultFieldKey.ts

```diff
@@ -3,8 +3,10 @@ enum DefaultFieldKey {
   CELL_NUMBER = "cell_number",
   CHILDREN = "children",
   CURRENT_CLASS = "current_class",
```

Comment:
nit: since we're adding `CURRENT_PREFERRED_CLASS`, let's rename this (should apply for the backend too)
```suggestion
  CURRENT_ENROLLED_CLASS = "current_enrolled_class",
```

Reply:
not quite sure how to make this happen in the backend

Reply:
oh that comment was super outdated haha, let's just rename it on the frontend